### DISCUSSION TITLE
Room navigation and take pictures in front of scenes

### DISCUSSION THREAD

**Discussion by yqi19** [allenai/ai2thor#1035] [General]
Hi, thanks for the amazing simulator!
We are working on ai2thor to collect dataset now, and I want to find out that how can I guide the agent to navigate around each scene, and take pictures in front of different scenarios.
Is there some quick way to solve this?
Thanks!

**Comment by mattdeitke** [comment]
Hey @yqi19,

Yes, this can be done :)

Can you try looking at getting frames from the environment: https://ai2thor.allenai.org/ithor/documentation/environment-state
And the navigation actions: https://ai2thor.allenai.org/ithor/documentation/navigation

You may also want to use GetReachablePositions and teleport the agent.

Hope that helps,
Matt :)
